changelog/extern_cpp_overhaul.dd

```diff
@@ -0,0 +1,24 @@
+`extern (C++)` construction, destruction, operators and other mangling improvements
+
+Many improvements have been made to the `extern(C++)` experience in this release cycle.
+
+Mangling improvements include:
+* Constructor/destructor mangling matches C++
+* Compatible D style operators now mangle as C++ operators
+* `nullptr_t` mangles correctly
+* Also various mangling bugs have been fixed
```

Comment:
This isn't Markdown, it's DDoc which for the time being doesn't support Markdown.

Reply:
I fixed this for you.